addons/packages/pinniped/0.12.1/bundle/config/overlay/post-deploy-overlay.yaml

```diff
@@ -54,6 +55,7 @@
 #@ commands.append("--is-dex-required={}".format(is_dex_required()))
 #@ commands.append("--concierge-is-cluster-scoped=true")
 
+#@ if render_on_mgmt_cluster() or render_on_workload_cluster():
```

Comment:
Just realizing that I don't think we have a final story to not deploy the post-deploy job on workload clusters after we're done removing any relevant workload cluster stuff from the post deploy job... maybe I'll make a note of it in this story (https://jira.eng.vmware.com/browse/TKG-10498)?

Reply:
Yeah I didn't read that in the AC of this story so did not cover that issue.
Combine or spin out another story?

Reply:
Think we covered this in `daily`. 👍